src/Hooks.cpp

```
@@ -31,31 +31,34 @@ namespace Hooks
-				RE::TESNPC* actorBase = skyrim_cast<RE::TESNPC*>(actor->data.objectReference);
-				uint32_t numHeadParts = 0;
-				RE::BGSHeadPart** Headparts = nullptr;
-
-				if (actorBase->HasOverlays()) {
-					numHeadParts = actorBase->GetNumBaseOverlays();
-					Headparts = actorBase->GetBaseOverlays();
-				} else {
-					numHeadParts = actorBase->numHeadParts;
-					Headparts = actorBase->headParts;
-				}
+			auto* userData = a_skeleton->GetUserData();
```

Clang (14.0.6)

[warning] variable 'userData' is not initialized

(cppcoreguidelines-init-variables)